src/Take.Blip.Builder/Utils/VariableReplacer.cs

```diff
@@ -1,14 +1,20 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.Text;
 using System.Text.RegularExpressions;
 using System.Threading;
 using System.Threading.Tasks;
+using Take.Blip.Builder.Hosting;
 
 namespace Take.Blip.Builder.Utils
 {
     public class VariableReplacer : IVariableReplacer
     {
-        private static readonly Regex TextVariablesRegex = new Regex(@"{{([a-zA-Z0-9\.@_-]+)}}", RegexOptions.Compiled);
+        private readonly Regex TextVariablesRegex;
```

Comment:
This regex it's simple and compiled, its okay to not add a timeout

Reply:
Done